app/views/course/admin/assessment_settings/edit.html.slim

```diff
@@ -0,0 +1,53 @@
+= page_header
+
+h2
+  span = t('.categories_and_tabs')
+  div.pull-right
+    - if can?(:create, Course::Assessment::Category)
+      = new_button([current_course, :admin, :assessments, :category])
+
+= simple_form_for(current_course, url: course_admin_assessments_path(current_course)) do |f|
```

Comment:
same thing for simple_form_for and simple_fields_for -- should we use the method version of the DSL version? I believe the DSL version was more prevalent.